### PR TITLE
Update REMIXD tutorial

### PR DESCRIPTION
**Pull request by smiasojed**
Closes https://github.com/paritytech/contract-issues/issues/7

### CODE REVIEW

**Comment by smiasojed** [comment]
> I think we should remove the quick dapp and just use REMIX to show how to compile a contract. But not how to build a Dapp.

I can remove this, but this PR is more about what to do with the local development section
 
> Because right now it is confusing because we show people two different ways to create a Dapp. We should prepare a template repository that people can clone to follow the ether.js part that sets them up with a basic Dapp.

REMIX supports workspaces from GitHub repository or Gist, which allows us to use them as the default workspace for tutorial

**Comment by smiasojed** [comment]
> Yes the part about local **contract** development is good. I just don't want to cover **frontend** development here.

I think that local development could be moved outside the tutorial. If we plan to work on template repo, REMIX should probably clone workspace directly from it.

The content which is not needed for the tutorial like quick dapp or local development could be moved to separate page or deleted if we do not see a value there.

**Comment by athei** [comment]
The point is to guide people to a workflow to iterate on their contract and frontend. Having the contract in the local file system would be good for that, right? I don't expect people to be keen on editing code inside the browser.

> REMIX should probably clone workspace directly from it.

That repo would be concerned with building the frontend. And I don't want to conflate REMIX with frontend development.

**Comment by smiasojed** [comment]
OK, In this case we need to start tutorial from cloning the template repo and connecting to local fs from REMIX

**Comment by smiasojed** [comment]
As discussed, we will keep only the basics in the Remix tutorial, without covering DApp deployment or development on the local filesystem.

File: docs/tutorial/remix-tips.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+## Connect to Your Local Filesystem
+
+If you prefer working with contracts on your local filesystem, Remix can connect directly to it. Keep in mind that connecting to your local filesystem will switch your current workspace. If you want to keep working on contracts from your current workspace, make sure to download them first.
+
+![Download workspace](img/download.png)
+
+Note that switching to a local filesystem will cause all previously deployed contracts, both pinned and unpinned, to become invisible in the **Deploy & Run** tab. If you need to interact with already deployed contracts, make sure to note down their addresses beforehand so you can load them later using the **At Address** button.
+
+![Load contract](img/load_contract.png)
```
Comment (athei) [inline docs/tutorial/remix-tips.md:45]: I think we should keep this in the main tutorial. Telling people how to interact with pre-existing contracts is basic knowledge. Just as a separate main heading "## How to interact with already deployed contracts".

Comment (smiasojed) [reply docs/tutorial/remix-tips.md]: done

**Comment by kianenigma** [comment]
> I think we should remove the quick dapp and just use REMIX to show how to compile a contract. But not how to build a Dapp.

Seconding this via #27.

**Review by athei — APPROVED** [review]

**Pull request merged**